.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Test
       run: go test -covermode atomic -coverprofile='${{ matrix.os }}-${{ matrix.go-version }}.cov' ./...
     - name: Coveralls Parallel
-      uses: coverallsapp/github-action@643bc377ffa44ace6394b2b5d0d3950076de9f63
+      uses: coverallsapp/github-action@v2
```

Comment:
We should probably try and pin to hash, dependabot supports those bumps.

Also, 643bc377ffa44ace6394b2b5d0d3950076de9f63 is the latest release commit AFIACT? https://github.com/coverallsapp/github-action/releases/tag/v2.3.0

So, do we need this diff?

Reply:
Mh, I missed this. In this case we do not need this PR. The fix on the dashboard fixed the coverall problems